src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -1432,8 +1432,15 @@ class MavlinkStreamVFRHUD : public MavlinkStream
 		updated |= _airspeed_sub->update(&_airspeed_time, &airspeed);
 
 		if (updated) {
-			mavlink_vfr_hud_t msg = {};
-			msg.airspeed = airspeed.indicated_airspeed_m_s;
+			mavlink_vfr_hud_t msg{};
+
+			if (hrt_elapsed_time(&airspeed.timestamp) < 1_s) {
+				msg.airspeed = airspeed.indicated_airspeed_m_s;
+
+			} else {
+				msg.airspeed = NAN;
```

Comment:
@dagar airspeed = NAN is displayed as 0.0m/s in QGC, correct?

Reply:
Yes.

![image](https://user-images.githubusercontent.com/84712/70456443-180d1980-1a7c-11ea-9393-3125e6afd132.png)

![image](https://user-images.githubusercontent.com/84712/70456453-1d6a6400-1a7c-11ea-9f43-c1014169f6d4.png)